File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/PaymentProduct5402SpecificOutputMapper.cs
using PaymentProduct5402SpecificOutputSdk = OnlinePayments.Sdk.Domain.PaymentProduct5402SpecificOutput;
using PaymentProduct5402SpecificOutputDto = Business.Domain.Payments.PaymentDetails.PaymentProduct5402SpecificOutput;

namespace Infrastructure.Mappers.PaymentDetails;

public static class PaymentProduct5402SpecificOutputMapper
{
    public static PaymentProduct5402SpecificOutputDto Map(PaymentProduct5402SpecificOutputSdk? response)
    {
        return new()
        {
            Brand = response?.Brand
        };
    }
}

File: server/php/src/Infrastructure/Mappers/PaymentDetails/PaymentProduct5402SpecificOutputMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\PaymentProduct5402SpecificOutput;

final class PaymentProduct5402SpecificOutputMapper
{
    public static function mapFromSdkResponse(?PaymentProduct5402SpecificOutput $response): ?PaymentProduct5402SpecificOutput
    {
        if ($response === null) {
            return null;
        }

        return new PaymentProduct5402SpecificOutput(
            brand: $response->Brand ?? null
        );
    }
}
